docs/source/developer/interactives.rst

```diff
@@ -146,13 +103,10 @@ Items of interest from this diagram:
   Any third-party libraries you include must have a licence that allows them to be used in the CSFG.
   A summary of the library's licence must be added to ``LICENCE-THIRD-PARTY`` with the full licence file added to ``cs-field-guide/third-party-licences/``.
 
-- ``static/interactives/<interactive-slug>/README.md`` - This is a short file that that explains the interactive's purpose and links to any necessary wiki pages.
-
-- ``static/interactives/<interactive-slug>/thumbnail.png`` - ``whole-page`` interactives required a thumbnail image showing the interactive in action.
-  Unless specified when included in the chapter ``md`` file, the file needs to be named as ``thumbnail.png``.
-  The image should be at least 900px wide, and of moderate quality (small size is more important than quality).
+- ``static/interactives/<interactive-slug>/README.md`` - This is a short file that that explains the interactive's purpose.
+  It can also include links to any documentation or websites that future developers may find useful.
 
-- ``templates/interactives/<interactive-interactive>.html`` - This is the HTML template for the interactive for a specific locale, as defined in the interactive configuration file.
+- ``templates/interactives/<interactive-slug>.html`` - This is the HTML template for the interactive for a specific locale, as defined in the interactive configuration file.
   Locale may or may not share a template for an interactive, allowing different versions of an interactive to be available (`see GitHub for more information <https://github.com/uccser/cs-field-guide/pull/710>`_).
```

Comment:
I don't see how that PR adds any more information

Reply:
Is there much harm in leaving it there?

Reply:
It's fine, I've changed my mind